api/http/router.go

```diff
@@ -38,6 +38,8 @@ const (
 	URITenantDevices = "/tenants/:tenant_id/devices"
 	URITenantDevice  = "/tenants/:tenant_id/devices/:device_id"
 
+	URIConfiguration = "/config/devices/:device_id"
```

Comment:
I would suggest naming this endpoint `configurations`, to match our naming convention in other services (e.g. `deployments`, `images`, no abbreviation and plural).

Reply:
changed.